docs/en/ingest-management/commands.asciidoc

```diff
@@ -452,3 +615,39 @@ elastic-agent version
 ++++
 <hr>
 ++++
+
+[discrete]
+[[elastic-agent-watch-command]]
+== elastic-agent watch
+
+Watch the {agent} for failures and initiate rollback.
+
+//QUESTION: When is this command used? During testing? 
```

Comment:
@michalpristas can confirm, but this is used by Agent itself during Agent upgrades, and *possibly also during initial install, but maybe not.

Reply:
I don't know if it has much value to users... if it does, I'd love to learn it and how to use it manually.  I believe that it starts up another process (a 'watcher' elastic agent process) which should go away when the upgrade successfully finishes and some 10-15 minute window is passed, or if the upgrade is confirmed to have failed.  It may be worthwhile to put into the docs somewhere (maybe not here) to expect this process to exist during Agent upgrades, for this purpose.

Reply:
I'm going to comment this out for now until we decide if it's worth exposing to end users.